dumux-precice/couplingadapter.cc

```diff
@@ -133,7 +100,7 @@ void CouplingAdapter::finalize()
         precice_->finalize();
 }
 
-double CouplingAdapter::advance(const double computedTimeStepLength)
+void CouplingAdapter::advance(const double computedTimeStepLength)
 {
     assert(wasCreated_);
     return precice_->advance(computedTimeStepLength);
```

Comment:
`advance` does not return anything now, so we have to do the same as we did for `initialize`above.